index.qmd

```diff
@@ -1,55 +1,224 @@
-[Get Started]({{< meta get-started >}}){.btn-action-primary .btn-action .btn .btn-lg role="button" style="background-color: #073c44; color: white"}
 :::
 
-::: {.g-col-3}
-:::
-:::
-:::
+[See all news &rarr;](news/)
 
-:::
+```{=html}
+<h3 class="pb-3 section-start-space">
+  Learn more
+</h3>
+<div class="list grid quarto-listing-cols-3">
+  <a class="pseudolisting" href="/library">
+    <div class="card-title h5">
+      Ecosystem
+    </div>
+    <p class="quarto-grid-item">
+      Learn about all the packages that are part of The Turing Organization.
+    </p>
+  </a>
+  <a class="pseudolisting" href="https://turinglang.org/docs/tutorials/docs-00-getting-started/">
+    <div class="card-title h5">
+      Documentation
+    </div>
+    <p class="quarto-grid-item">
+      Familiarize yourself with the Turing.jl API.
+    </p>
+  </a>
+  <a class="pseudolisting" href="https://turinglang.org/docs/tutorials/00-introduction/">
+    <div class="card-title h5">
+      Official tutorials
+    </div>
+    <p class="quarto-grid-item">
+      Learn to tackle specific problems with Turing.jl.
+    </p>
+  </a>
+</div>
+```
 
-::: {style="position: relative; display: flex; justify-content: center;"}
-::: {.grid}
-::: {.g-col-1}
-:::
-::: {.g-col-8}
-## Community Resources
+::: {.section-start-space}
 
-The Turing.jl [discourse forum](https://discourse.julialang.org/c/domain/probprog) provides support and discussion for all user levels.
-The [Turing.jl documentation]({{< meta get-started >}}) and [official tutorials]({{< meta tutorials-intro >}}) help users learn and use Turing.jl!
-There's also a [tutorial in Bayesian Statistics using Julia and Turing.jl](https://storopoli.io/Bayesian-Julia).
+### Algorithms
+
+Placeholder text introducing this section on algorithms.
+
+| Column 1 | Column 2 | Column 3 | Column 4 |
+|----------|----------|----------|----------|
+|  Entry A |  Entry D |  Entry G |  Entry J |
+|  Entry B |  Entry E |  Entry H |  Entry K |
+|  Entry C |  Entry F |  Entry I |  Entry L |
+
+: Supported MCMC and other learning algorithms {.striped .borderless}
 
 :::
-::: {.g-col-3}
-:::
-:::
+
+::: {.section-start-space}
+
+### Bayesian Workflow
+
+Placeholder text introducing the Bayesian Workflow diagram from the ACM special issue submission.
+
+```{=html}
+<div style="text-align:center;">
+<img src="https://i0.wp.com/www.honeybeesuite.com/wp-content/uploads/2019/09/Multiple-teeth-HBS.jpg?w=400&ssl=1" style="margin:0.5rem 0;border-radius:10pt">
+<p>An example of the Beeysian workflow.</p>
+</div>
+```
+
 :::
+
+```{=html}
+<div class="d-flex flex-row panel-wrapper gap-3 section-start-space">
+<div class="flex-grow-1">
+<h3>Talks</h3>
+  <p>Placeholder for intro text on list of talks which are available online</p>
+  <ul>
+    <li>Talk 1
+    <li>Talk 2
+    <li>Talk 3
+  </ul>
+</div>
+<div class="flex-grow-1">
+  <h3>Inofficial tutorials</h3>
+  <p>Placeholder for intro text on list of inofficial tutorials</p>
```

Comment:
```suggestion
  <h3>Other resources</h3>
  <p>This list contains a few other written resources for learning Turing. If you have written something and would like to share it with the community, please <a href="https://github.com/TuringLang/docs/issues/new">get in touch</a>!</p>
```

Reply:
Not sure if the docs repo is the best place to raise an issue. I figured it would either be the docs repo or this repo, but this one is quite low-profile so it might not get noticed very soon.